app-admin/filebeat/files/filebeat-glibc-2.34.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/libbeat/common/seccomp/policy_linux_386.go b/libbeat/common/seccomp/policy_linux_386.go
```

Comment:
When you can, pop a link to the upstream bug/commit(s) at the top of patches

Reply:
I put it into the commit message, but it can be here too.